config/api.yml

```diff
@@ -621,6 +621,12 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :ems_events:
```

Comment:
@rodneyhbrown7 on second thought, I think this change should be removed from this PR.  I don't think you need to expose the events over the API for now.

If you guys decide later to expose events over the API, we should chat about the use case and how they're going to be used.  For now, events are typically handled by either the Automate Engine, or by Control/Policy.